assets/i18n/cy.json

```
@@ -35,7 +35,10 @@
   "ONLINE": "Ar-lein",
   "ASK_HMRC_ONLINE": "Gofyn i CThEF ar-lein",
   "WE_MAY_CALL_YOU": "Mae’n bosibl y byddwn yn eich ffonio am eich hawliad os oes gennych broblem.",
+  "MAKE_A_CLAIM_AFTER_24_HRS": "Rydych wedi cyflwyno hawliad. Gallwch wneud hawliad arall ar ôl 24 awr",
   "BEGIN_NEW_CLAIM": "Dechrau hawliad newydd",
+  "HAVE_EXISTING_CLAIM_IN_PROGRESS": "You have an existing claim in progress",
+  "CANNOT_START_A_NEW_CLAIM": "You cannot start a new claim until you have completed the current one",
```

Should this be in Welsh on the AC's for the US-12640 translation is item 6

---

```suggestion
  "HAVE_EXISTING_CLAIM_IN_PROGRESS": "Mae hawliad ar y gweill gennych.",
  "CANNOT_START_A_NEW_CLAIM": "Ni allwch ddechrau hawliad newydd nes eich bod wedi cwblhau’r hawliad presennol.",
```

Coming from the welsh translation excel in the ticket.

---

we have got the welsh translations today, so have added it now